index.html

```diff
@@ -982,31 +982,30 @@ <h2>
         };
       </pre>
       <p>
-        <a>PushMessageData</a> objects have an associated [=byte sequence=] set on creation, which
-        is `null` if there was no data in the <a>push message</a>.
+        <a>PushMessageData</a> objects have an associated <dfn data-dfn-for=PushMessageData>bytes</dfn> (a
```

Comment:
Is `data-dfn-for` needed when `<section>` already have it above?

Reply:
I don't know. I could try I suppose. Is it r+ with that and the below nit?

Reply:
Does that apply it to both the `<dfn>` and the `<a>`s in that section? So if you reference outside you'd have to technically override it?